pkg/plugins/pluginrpc-gen/README.md

```diff
@@ -30,7 +30,7 @@ Where:
 - `--type` is the name of the interface to use
 - `--name` is the subsystem that the plugin "Implements"
 - `-i` is the input file containing the interface definition
-- `-o` is the output file where the the generated code should go
```

Comment:
Thanks for your work. Actually we usually do not change the code under directory of vendor, like file `vendor/github.com/davecgh/go-spew/spew/format.go` is in directory vendor. So, please revert those parts in your pr.

If we have to update those repo, we should make change in the upstream repo, for example repo go-spew in above sample. then use hack/vendor.sh to revendor that.